python/labbox_ephys/api/_workersession.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, *, labbox_config):
         self._on_messages_callbacks = []
         self._queued_document_action_messages = []
         self._additional_subfeed_watches = []
+        self._workspace_subfeed_watches = []
```

Comment:
Special subfeed watches that provide updates to the client when the workspace subfeed is modified.